api/tests/routes/backoffice/offers_test.py

```diff
@@ -1530,6 +1530,45 @@ def test_list_offers_filter_on_price(self, authenticated_client):
         rows = html_parser.extract_table_rows(response.data)
         assert len(rows) == 0
 
+    def test_list_offers_filter_on_offerer(self, authenticated_client):
+        venue1 = offerers_factories.VenueFactory()
+        venue2 = offerers_factories.VenueFactory(managingOfferer=venue1.managingOfferer)
+        offerers_factories.VenueFactory()
+        offers_factories.OfferFactory(name="hide")
+
+        query_args = {
+            "algolia_search": "",
+            "limit": 1000,
+            "search-0-search_field": "OFFERER",
+            "search-0-operator": "IN",
+            "search-0-offerer": venue1.managingOffererId,
+        }
+
+        with patch("pcapi.routes.backoffice.offers.blueprint.search_offer_ids", return_value=[]) as algolia_mock:
+            # one more request to get the venues from offerer id
+            # one more request to cherck the offerer id
+            # no offers from algolia therefore no request to retrieve their fields
+            with assert_num_queries(self.expected_num_queries + 1):
+                response = authenticated_client.get(url_for(self.endpoint, **query_args))
+                assert response.status_code == 200
+
+            # handle venue order randomness to avoid flakyness
+            try:
+                algolia_mock.assert_called_once_with(
+                    query="",
+                    filters=f"(venue.id:{venue1.id} OR venue.id:{venue2.id})",
+                    count=1001,
+                )
+            except AssertionError:
+                algolia_mock.assert_called_once_with(
+                    query="",
+                    filters=f"(venue.id:{venue2.id} OR venue.id:{venue1.id})",
+                    count=1001,
+                )
+
+        rows = html_parser.extract_table_rows(response.data)
+        assert len(rows) == 0
```

Comment:
Suggestion : plus directement : 
```suggestion
        assert html_parser.count_table_rows(response.data) == 0
```